examples/performance/include/traccc/efficiency/seeding_performance_writer.hpp

```diff
@@ -70,6 +61,16 @@ class seeding_performance_writer {
         }
     }
 
+    void init() {
+        auto path = m_cfg.file_path;
```

Comment:
Is there any reason for making this copy? :thinking: Why not just refer to `m_cfg.file_path` in the 2 places in this function?

Even if you'd like to make the variable name shorter, please declare the type explicitly as

```c++
const std::string&
```

instead of using `auto`.

Reply:
I will change to `const std::string&`.